htetl/extract/phones.py

```diff
@@ -50,34 +44,20 @@ def _norm(text):
 
     return '-'.join([norm_area_code, norm_first_three, norm_last_four])
 
-class ParsePhones(luigi.Task):
+
+def extract_phone(df):
```

Comment:
Looks okay. Nothing too crazy; just changing the job from a luigi task to a function for the main job